core/src/test/scala/io/qbeast/core/model/CubeIdTest.scala

```diff
@@ -23,6 +23,19 @@ class CubeIdTest extends AnyFlatSpec with Matchers {
     id1 == id2 shouldBe false
     id3 == id4 shouldBe false
     id1 == id5 shouldBe false
+    val id6 =
```

Comment:
Probably a bit overkilling, but maybe a few more tests would be good to be sure we are covering all the cases: when the trim aligns with a long, when it is just a bit shorter or just a bit longer.